sdk/keyvault/azure-security-keyvault-jca/src/main/java/com/azure/security/keyvault/jca/KeyVaultTrustManager.java

```diff
@@ -32,10 +32,21 @@
     /**
      * Stores the default trust manager.
      */
-    private X509TrustManager defaultTrustManager;
+    private static final X509TrustManager DEFAULT_TRUST_MANAGER = defaultTrustManager();
 
     /**
-     * Stores the keystore.
+     * Stores the default trust manager with key vault key store.
```

Comment:
Can we just keep delete `kvTrustManager` and `otherTrustManager`, and make defaultTrustManager use the jreKeyStore?